### PR TITLE
Add column title for details column on the 'Traces' page

### LINKED ISSUE #1535: [WebToolsE2E][Aspire] On the dashboard, no column title for view column on the 'Traces' board.
**REGRESSION INFO**: Also repro on Aspire.ProjectTemplates.8.0.0-preview.1.23557.2

**INSTALL STEPS**

1. Clean machine: Win 11 23H2
2. Install latest VS 17.9 P3 from [34502.94.main]
3. CLI workload update and install aspire latest version: [Aspire.ProjectTemplates.8.0.0-preview.3.24052.6]
4. Apply NuGet Feeds

**REPRO STEPS**

1. In the VS, create a new project > .NET Aspire Starter Application > .NET 8.0 > Create.
2. Ctrl F5.
3. On the dashboard, open any endpoint.
4. Then click 'Traces'

**ACTUAL**
No column title for view column.
image

**EXPECTED**
Maybe it would be better to display 'Details' as title here, like under 'Structured logs' board.
image

### PR DESCRIPTION
**Pull request by stbau04**
Add column title for details column on the 'Traces' page

Fixes: https://github.com/dotnet/aspire/issues/1535

Traces:
grafik

Structured logs:
grafik

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/dotnet/aspire/pull/1574)

### CODE REVIEW

**Comment by stbau04** [comment]
FYI: it took me four runs to pass all checks
Once a unit test failed due to a kubeconfig that could not be deleted/modified, once the linux tests timed out and once the windows tests timed out. I don't think that the problems are related to my changes, are there any known issues with the pipeline?

File: src/Aspire.Dashboard/Components/Pages/Traces.razor [hunk 1]
```diff
@@ -76,7 +76,7 @@
                     </div>
                 </TemplateColumn>
 
-                <TemplateColumn>
+                <TemplateColumn Title="@ControlsStringsLoc[nameof(ControlsStrings.DetailsColumnHeader)]">
```
Comment (adamint) [inline src/Aspire.Dashboard/Components/Pages/Traces.razor:79]: Could you put this string in `TraceDetail.resx`?

File: src/Aspire.Dashboard/Components/Pages/StructuredLogs.razor [hunk 2]
```diff
@@ -92,7 +92,7 @@
                                     </a>
                                 }
                             </TemplateColumn>
-                            <TemplateColumn Title="@Loc[nameof(Dashboard.Resources.StructuredLogs.StructuredLogsDetailsColumnHeader)]">
+                            <TemplateColumn Title="@ControlsStringsLoc[nameof(ControlsStrings.DetailsColumnHeader)]">
```
Comment (adamint) [inline src/Aspire.Dashboard/Components/Pages/StructuredLogs.razor:95]: Could you put this string in `StructuredLogs` instead of `ControlsStrings`?

**Review by adamint — CHANGES_REQUESTED** [review]

**Comment by adamint** [comment]
> are there any known issues with the pipeline 

Not AFAIK @stbau04, let me know if you continue to run into this pipeline issue.

**Review by adamint — APPROVED** [review]

**Review by JamesNK — APPROVED** [review]

**Comment by JamesNK** [comment]
Thanks!

**Pull request merged**